src/main/java/frc/robot/RobotMap.java

```diff
@@ -0,0 +1,24 @@
+package frc.robot;
+
+public class RobotMap {
+    /** Driving Motor CAN IDs  */
+    public static final int FRONT_LEFT_DRIVE_MOTOR_CAN_ID = 1,
+     FRONT_RIGHT_DRIVE_MOTOR_CAN_ID = 3,
+     BACK_LEFT_DRIVE_MOTOR_CAN_ID = 2,
+     BACK_RIGHT_DRIVE_MOTOR_CAN_ID = 4;
+
+    /** Steering Motor CAN IDs */
+    public static final int FRONT_LEFT_STEERING_MOTOR_CAN_ID = 5,
+     FRONT_RIGHT_STEERING_MOTOR_CAN_ID = 6,
+     BACK_LEFT_STEERING_MOTOR_CAN_ID = 7,
+     BACK_RIGHT_STEERING_MOTOR_CAN_ID = 8;
```

Comment:
I'm not sure if this was on purpose, but I noticed that for the DRIVE_MOTOR_CAN_IDs the front IDs are both odd numbers and the back IDs are both even numbers. This is not the case for the STEERING_MOTOR_CAN_IDs.

Am I misinterpreting the numbering system for the CAN_IDs or was it just random/coincidence?

Reply:
I'm pretty sure it was random/coincidence but now I'm wondering: if the Drive Motor Can ID's are both odd and the backs are both even, does that have to be the same with the steering motors? Or does it not matter? If it does matter then I can change it so that the steering motors follow the same pattern.

Reply:
I'm not exactly sure ... I thought we tried having a numbering convention last year but it just got too complicated (I could be mistaken though). @tails618 thoughts?